config/locales/simple_form.en.yml

```diff
@@ -13,6 +13,13 @@ en:
           url: Fedora URL should not end with a slash
         name: A single or hyphenated name used for technical aspects of the repository (e.g., "acme" or "acme-library").
         gtm_id: The ID of your Google Tag Manager account
+        analytics_id: The ID of your Analytics account
+        analytics_oauth_app_name: The name of the application
+        analytics_oauth_app_version: The version of application
+        analytics_oauth_private_key_secret: The secret provided when you created the key
```

Comment:
can you review these last 3 with the idea of "by who and where?" added in? there is a lot implied by these as terse as they are.

Reply:
> by who

I pulled these from what was in the readme and left them vague since this work is going back to hyku, which can also use matomo. and potentially other applications in the future? 

> and where

I wasn't sure where these things could be found for matomo. 
<hr>

would you prefer I explicitly state google and matomo for now? and look up where these things can be found for matomo?

Reply:
ref: #492